client/src/Google_API/calendar_event.jsx

```diff
@@ -3,11 +3,11 @@
 // Change the scope to 'https://www.googleapis.com/auth/calendar' and delete any
 // stored credentials.
 
-
+import axios from 'axios';
 // SET UP GOOGLE ENVIRONMENT
 var gapi = window.gapi;
-var CLIENT_ID = "288705266674-o6jgvsnd9frg0slcqgdknsr91q14im54.apps.googleusercontent.com";
```

Comment:
since these have been pushed to Github, we will need to get an updated `CLIENT_ID` and `API_KEY` and stop using these for security purposes